src/Sylius/Bundle/CoreBundle/Fixture/Factory/OrderExampleFactory.php

```diff
@@ -318,4 +327,34 @@ protected function setOrderCompletedDate(OrderInterface $order, \DateTimeInterfa
             $order->setCheckoutCompletedAt($date);
         }
     }
+
+    protected function fulfillOrder(OrderInterface $order): void
+    {
+        $this->applyPaymentCompleteStateTransition($order);
+        $this->applyShipmentShipStateTransition($order);
+    }
+
+    protected function applyPaymentCompleteStateTransition(OrderInterface $order): void
+    {
+        /** @var PaymentInterface $payment */
+        $payment = $order->getPayments()->first();
```

Comment:
For more future proof solutions, you should iterate over this collection and complete all of the payments that can be completed.